### PR TITLE
test(probe): add protocol probing test

### PR DESCRIPTION
**Pull request by whalecold**
#### What type of PR is this?

test 
#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.
- [x] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional):

### CODE REVIEW

File: code/kitex_sample/gen-go/hello/api.go [hunk 1]
```diff
@@ -0,0 +1,1061 @@
+// Code generated by thriftgo (0.3.12). DO NOT EDIT.
```
Comment (DMwangnima) [inline code/kitex_sample/gen-go/hello/api.go:1]: What does this generated file do?

File: code/kitex_sample/handler/handler.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+	"github.com/kitex-contrib/codec-dubbo-tests/code/kitex_sample/kitex_gen/hello"
+)
+
+// GreetServiceImpl implements the last service interface defined in the IDL.
+type GreetServiceImpl struct{}
+
+// Greet implements the GreetServiceImpl interface.
+func (s *GreetServiceImpl) Greet(ctx context.Context, req string) (resp string, err error) {
+	time.Sleep(time.Millisecond * 10)
```
Comment (DMwangnima) [inline code/kitex_sample/handler/handler.go:34]: The reason for sleep?

Comment (whalecold) [reply code/kitex_sample/handler/handler.go:34]: the code is used for test, i will delete it.

File: code/kitex_sample/gen-go/hello/api.go (hunk 1 above)
Comment (whalecold) [reply code/kitex_sample/gen-go/hello/api.go:1]: mistake commit~

File: code/kitex_sample/handler/handler.go (hunk 2 above)
Comment (whalecold) [reply code/kitex_sample/handler/handler.go]: Fixed

**Review by DMwangnima — APPROVED** [review]

**Pull request merged**